### PR TITLE
Update site description with mission statement

### PR DESCRIPTION
**Pull request by valencik**
The mission statement taken from:
https://typelevel.org/blog/2022/04/01/call-for-steering-committee-members.html

image

### CODE REVIEW

**Comment by valencik** [comment]
Recently I heard someone unfamiliar with our community try to summarize Typelevel as a group that makes compiler plugins.
We're certainly a lot more than that, and I think this mission statement from the call for committee members does a better job introducing Typelevel as a community.

**Comment by valencik** [comment]
Since this is so front and center, I'm going to tag the whole group @typelevel/steering :)

File: _config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # ---------------------------------------------------------------
 name: Typelevel
 title: Typelevel
-description: Let the Scala compiler work for you. We provide type classes, instances, conversions, testing, supplements to the standard library, and much more.
+description: Typelevel is an association of projects and individuals united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.
```
Comment (armanbilge) [inline _config.yml:5]: Sounds more "peopley" than "individuals" :)

```suggestion
description: Typelevel is an association of projects and people united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.

```

Comment (armanbilge) [inline _config.yml:5]: idk. "association" sounds so formal, like The Typelevel Association. Are we an association?

```suggestion
description: Typelevel is an group of projects and individuals united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.

```

Comment (valencik) [reply _config.yml:5]: Combining a bit with your other comment, what about:

> Typelevel is a community of people and projects united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.

Comment (valencik) [reply _config.yml:5]: Actually "community of people" is kind of redundant, and you can't really have a community of projects...
:thinking:

Comment (valencik) [reply _config.yml:5]: > Typelevel is an ecosystem of projects and community united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.

???

Comment (SethTisue) [reply _config.yml]: idk what's best, but perhaps "Typelevel is a collective dedicated to..."

**Review by jducoeur — APPROVED** [review]
I might leave off the "community of", but I'm of mixed minds -- this works reasonably well, and is a clear improvement IMO.

**Review by samspills — APPROVED** [review]

**Review by armanbilge — APPROVED** [review]

File: _config.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # ---------------------------------------------------------------
 name: Typelevel
 title: Typelevel
-description: Let the Scala compiler work for you. We provide type classes, instances, conversions, testing, supplements to the standard library, and much more.
+description: Typelevel is an ecosystem of projects and community of people united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.
```
Comment (JasnaMRB) [inline _config.yml:5]:
```suggestion
description: Typelevel is an ecosystem of projects and a community of people united to foster an inclusive, welcoming, and safe environment around functional programming in Scala.
```
optional

**Review by JasnaMRB — APPROVED** [review]

**Comment by valencik** [comment]
> This is of course already mergeable as a major improvement, yet I can't resist the urge to brainstorm more variants. Maybe:
> 
> > Typelevel is a community working together since 2013 to foster an inclusive, welcoming, and safe environment around functional programming in Scala. Our ecosystem of libraries is mature and widely adopted.

@SethTisue I like that expanded version. It makes me think I should follow this up with an edit to the actual About page and provide some more details/context. I think I'll keep the "description" tagline this shorter version though.
Thanks! (I'll be sure to ping you when I open the other PR :)

**Pull request merged**